RevenueCatUI/PaywallView.swift

```diff
@@ -350,7 +356,11 @@ struct LoadedOfferingPaywallView: View {
         if self.displayCloseButton {
             NavigationView {
                 view
-                    .toolbar { self.toolbar }
+                    .toolbar {
+                        self.makeToolbar(
```

Comment:
I'm wondering, does this mean it will recreate the toolbar every time it needs to redraw? Do we need to mark `makeToolbar` with the `@ViewBuilder` annotation? (Not superfamiliar with swiftUI so it might be able to optimize it somehow...)

Reply:
This can't be `@ViewBuilder` because weirdly enough, `ToolbarContent` isn't a `View` 😅  This will recreate the toolbar content object but won't necessarily redraw the UI since SwiftUI does those diffs under the hood